src/components/Login/GoogleLogoutBtn.js

```diff
@@ -32,20 +33,27 @@ const GoogleLogoutBtn = ({
     unsetUser();
     logoutSuccess("Successfully logged out!");
     closeModal();
+    return response;
```

Comment:
@abid-salahi 

Notes: There was a bug from a previous commit whereby if a user logs in with a gmail account through the normal login (not the google one), they are given the google logout, which then didn't work. The issue persists in this commit. (Copied straight from my commit description because I'm lazy xD). Not sure how to fix this, so I'll pass it back to you

Not sure if the response is supposed to do anything. I logged it and it gives "Undefined", but to get rid of the lint warning, I just returned the response xD.

Reply:
yeah its a useless response. Could even remove it.